v2/fleet-local/control/mesos-master@.service

```diff
@@ -26,7 +26,7 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
   --log_dir=/var/log/mesos \
   --quorum=3 \
   --work_dir=/var/lib/mesos/master \
-  --zk=zk://${ZOOKEEPER_ENDPOINT}/mesos"
+  --zk=zk://`etcdctl get /environment/ZOOKEEPER_ENDPOINT`/mesos"
```

Comment:
for the most part, I believe that shell-outs in bash `-c` commands are backticks. but still, noted. It'll be one of the things that we standardize when we clean this a bit more.